packages/core/src/modules/interest-group/interest-group-repository.ts

```diff
@@ -28,9 +30,21 @@ export class InterestGroupRepositoryImpl implements InterestGroupRepository {
     return interestGroup ? mapToInterestGroup(interestGroup) : undefined
   }
 
-  async getAll(pageable: Pageable): Promise<Collection<InterestGroup>> {
-    const res = paginatedQuery(this.db.selectFrom("interestGroup").selectAll(), pageable, mapToInterestGroup)
-    return res
+  async getAll(pageable: Pageable) {
+    const query = this.db.selectFrom("interestGroup").selectAll()
+
+    const result = await singleColPaginatedQuery(query, {
+      pageable,
+      decodeCursor: decodeUlidIdCursor,
+      buildCursor: buildUlidIdCursor,
+      column: "id",
+      order: "desc",
```

Comment:
IMO DRF's API for ordering is quite nice, and I think we could have something similar https://www.django-rest-framework.org/api-guide/filtering/#specifying-which-fields-may-be-ordered-against

Reply:
What about just using the order of cols given to `column` when that option supports multiple columns and not just one like now

Reply:
fair, but how do you determine if the default sort is ASC or DESC?

Reply:
Passing order like this

Reply:
I think the pagination should be generic and support a list of columns, it seems rather easy to do. Whether you make the API be `["X", { columnName: "Y", ordering: ASCENDING/DESCENDING }]`, or `["X", "-Y"]` I'm somewhat indifferent about

The cursor can just be base64 of e.g. a JSON-array like `["columnAValue", "columnBValue"]`

Reply:
Yep, make cursor a base64 of some json object or array